### PR TITLE
cordentity

### PR DESCRIPTION
**Pull request by alkopnin**
Signed-off-by: Alexander Kopnin <alkopnin@gmail.com>

The covid cause to a high demand on digital identity technology. The Cordentity implements an efficient way to connect Indy and Corda so it makes possible to build sophisticated use cases where a verifier accepts credentials directly to a private decentralized network. The project was archived in 2019 due to low support, but in the mid/end of 2020 there is a number of external requests to unfreeze the project.

### CODE REVIEW

File: labs/cordentity.md [hunk 1]
```diff
@@ -1,8 +1,3 @@
----
-layout: default
-title: Cordentity
-parent: Archived Labs
----
```
Comment (tkuhrt) [inline labs/cordentity.md:5]: I believe these changes need to exist in order for https://labs.hyperledger.org/ to show appropriately. Change line 4 to `Labs`, but leave the other lines as is.

**Review by tkuhrt — CHANGES_REQUESTED** [review]
I am happy for this to be unarchived. Just minor changes requested.

File: labs/cordentity.md (hunk 1 above)
Comment (vipinsun) [reply labs/cordentity.md:5]: If there are external demands to unarchive, can you add any indication of who they are and how they will participate.

**Review by vipinsun — APPROVED** [review]
LGTM, always a pleasure to see something like this being unarchived.

**Comment by ryjones** [comment]
@vipinsun @tkuhrt check out #160

**Pull request closed without merging**